config/webpack/development.js

```diff
@@ -3,6 +3,6 @@ process.env.NODE_ENV = process.env.NODE_ENV || 'development'
 const environment = require('./environment')
 
 const config = environment.toWebpackConfig();
-config.output.filename = "js/[name]-[hash].js";
+config.output.filename = "packs/js/[name]-[hash].js";
 
-module.exports = environment.toWebpackConfig()
+module.exports = config;
```

Comment:
I see that we were exporting the original config instead of the changed config, so this change makes sense 👍